### PR TITLE
Added babel-transform-in-browser@0.1.0

### PR DESCRIPTION
**Pull request by voronianski**
https://github.com/voronianski/babel-transform-in-browser

> Transform ES2015 in browser on the fly with Babel.js for rapid prototyping

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Would you please add "repository" info? Thanks.

**Comment by PeterDaveHello** [comment]
I noticed that the popularity of this lib is not high enough, need a higher popularity to be hosted here.

**Comment by voronianski** [comment]
@PeterDaveHello that's strange, how do you check the popularity, is it quantity of GitHub stars? 

It will be very helpful to have this script on cdn, there were some [discussions](https://twitter.com/floydophone/status/680226147213426688) on how to start prototyping with React quickly, few notes here - https://github.com/voronianski/babel-transform-in-browser#quick-react-prototyping.

**Comment by PeterDaveHello** [comment]
@voronianski we usually take the stats on GitHub and npm registry as the signal of popularity, and it's required since it's impossible to host all the libs, we need a filter to filt out the libs really have some users, so that's meaningful to be hosted on a cdn.

**Comment by voronianski** [comment]
@PeterDaveHello but this kind of lib is not what will be downloaded from npm registry many times, it's real value starts when it's hosted on cdn (in order to copy paste link in html file)..

**Comment by PeterDaveHello** [comment]
@voronianski I understand what do you mean, but we still need a signal to prove its popularity, maybe you can give me different idea?

**Comment by pvnr0082t** [comment]
@PeterDaveHello Still not enough popularity. Should we close this pull request?

**Comment by PeterDaveHello** [comment]
Looks better now, let's add this one, @pvnr0082t would you please help review the content?

File: ajax/libs/babel-transform-in-browser/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "author": "Dmitri Voronianski <dmitri.voronianski@gmail.com>",
+  "license": "MIT",
+  "dependencies": {
+    "babel-standalone": "^1.0.0",
+    "currentscript": "^1.1.0"
+  },
+  "npmName": "babel-transform-in-browser",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
```
Comment (pvnr0082t) [inline ajax/libs/babel-transform-in-browser/package.json]: Let's use `"dist"` here.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by sufuf3** [comment]
Hi @voronianski , would you like to fix this PR and let it be merged ASAP? Thank you.

**Comment by sufuf3** [comment]
@PeterDaveHello this library doesn't have enough popularity.
And there is no response from the author since 11 Jun.
What can we do for this PR?
Just close or help finish this PR?
Thank you.

**Comment by PeterDaveHello** [comment]
Let's close this one.

**Pull request closed without merging**